src/plugins/intel_cpu/src/nodes/reduce.cpp

```diff
@@ -2922,6 +2922,23 @@ bool Reduce::canApplyJIT(const Precision &input_prec, const Precision &output_pr
            std::find(std::begin(supportedPrecisions), std::end(supportedPrecisions), output_prec) != std::end(supportedPrecisions);
 }
 
+size_t Reduce::getFusingAxis() const {
+    size_t channelAxis = 1;
+    if (!keep_dims) {
+        for (auto &raw_axis : raw_axes) {
+            int axis = raw_axis >= 0 ? raw_axis : raw_axis + static_cast<int>(getInputShapeAtPort(REDUCE_DATA).getRank());
+            if (axis == 1) {
```

Comment:
cast `-1` to negative value is not good, why not keep it as `-1` and change `size_t` to `int` ? Is there any condition, we cannot cover ?

Reply:
Fixed. Thanks Yi!